site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -164,6 +164,9 @@ include::_tenant-generator-type-config-rules.adoc[]
 [field]#tenant.externalIdentifierConfiguration.emailVerificationIdTimeToLiveInSeconds# [type]#[Integer]# [required]#Required# [since]#Available since 1.8.0#::
 The time in seconds until a email verification Id is no longer valid and cannot be used by the Verify Email API. Value must be greater than 0.
 
+[field]#tenant.externalIdentifierConfiguration.emailVerificationOneTimeCodeGenerator# [type]#[Integer]# [optional]#Optional# [since]#Available since 1.27.0#::
+A one time code used by the user to verify their email. This is used during a gated workflow. The values generated are `randomAlphaNumeric` and are 6 digits in length.
+
```

Comment:
The generator configurations have two properties, see `emailVerificationIdGenerator.type` and `emailVerificationGenerator.length` for an example. This will need to be documented with two fields. 

The description is pretty good, when using gated email verification, this value in conjunction with the high entropy `verificationId` will be used.